manager/controllers/advisories.go

```diff
@@ -91,7 +91,7 @@ func buildQueryAdvisories(account string) *gorm.DB {
 	return query
 }
 
-func buildAdvisoriesData(advisories []AdvisoriesDBLookup) *[]AdvisoryItem {
+func buildAdvisoriesData(advisories []AdvisoriesDBLookup) []AdvisoryItem {
```

Comment:
Why are we changing that to non-pointer?

Reply:
The same question for next 3 cases. Pointer usage should be more efficient because it avoids data copying.

Reply:
Slice is implemented as a fat pointer. The only thing copied is a pointer to data, length and capacity, 24 bytes instead of 8.